client/src/component/County/AuditBoard/Page.tsx

```diff
@@ -41,7 +41,8 @@ class AuditBoardSignInPage extends React.Component<PageProps, PageState> {
         } = this.props;
 
         const submit = () => {
-          auditBoardSignIn(auditBoardIndex, this.state.form);
+            auditBoardSignIn(auditBoardIndex, this.state.form);
+            window.history.pushState({}, 'login', '/county/audit/' + auditBoardIndex);
```

Comment:
Consider passing `history` down from the parent container component rather than accessing `window.history` directly

Reply:
why?

Reply:
- [Avoids problems with browsers handling `window.history` differently](https://reactarmory.com/guides/context-free-react-router#History-objects): 
- Avoids assumptions about the global state (e.g. tests using a non-browser environment)

Reply:
Or even better, [pass a callback down from the parent](https://github.com/democracyworks/ColoradoRLA/pull/116/commits/1ccb5ff57bc86f26691676da8916cd30c6759489)